apps/web/app/channels/notifications/page.tsx

```
@@ -0,0 +1,269 @@
+      {/* Header */}
+      <div
+        className="flex items-center justify-between px-4 py-3 border-b flex-shrink-0"
+        style={{ borderColor: "var(--theme-bg-tertiary)" }}
+      >
+        <span className="font-semibold text-white">Notifications</span>
+        {unreadCount > 0 && (
+          <button
+            onClick={markAllRead}
+            className="flex items-center gap-1.5 text-xs px-2 py-1 rounded transition-colors hover:bg-white/10"
+            style={{ color: "var(--theme-text-muted)" }}
+          >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)